libsrc/target/aquarius/far/l_far_mapaddr.asm

```diff
@@ -14,13 +14,11 @@ l_far_mapaddr:
     ld      a,e    ; If banked address is 0x00nnnn, then address absolutely
     and     a 
     ret     z
-    dec     e      ; __banked starts from 1
```

Comment:
No need to +1 or -1 as actual RAM page numbers are now used. Just need to shift the page number.